js/containers/landingsside/DinSituasjonContainer.js

```diff
@@ -7,25 +7,26 @@ import DinSituasjon from '../../components/landingsside/DinSituasjon';
 const { BEKREFTET, SENDT, TIL_SENDING } = statuser;
 const { ARBEIDSTAKER } = situasjoner;
 
-export function filtrerSykemeldingerPaaPeriode(sykmeldinger) {
+export function filtrerSykmeldingerPaaPeriode(state) {
     const treMndSiden = new Date();
     treMndSiden.setMonth(treMndSiden.getMonth() - 3);
 
-    return sykmeldinger.filter((sykmelding) => {
+    return state.dineSykmeldinger.data.filter((sykmelding) => {
```

Comment:
Dette er kanskje en selector?

Reply:
Ja, hva skal til for at den blir en selector da? At funksjonsnavnet inneholder ordet "selector"t? :)